### ISSUE TITLE
panos_lic timer

### ISSUE THREAD

**Issue by torstenstern** [PaloAltoNetworks/ansible-pan#183]
Good morning,

i play in the last time a lot with the ansible and i think that we need in this module a timer how will stop the script after the applying of the auth_codes was sucessful, becasue the FW will reboot automatically and i lost everytime the connection and the script will fail...off course and therefore would it be really helpful to have an timer in this script of 60 - 180 seconds.

**Comment by trustinger** [comment]
In the meantime, a reasonable workaround that Ive used for upgrading Juniper devices is to add a `wait_for` task after the reboot kicks off that keeps the rest of the playbook from failing. 
```
- name: Pause for reboot after upgrade
  wait_for:
    port: 22
    host: "{{ inventory_hostname }}"
    delay: 420
    timeout: 1500
    sleep: 5
```
This tells it to start checking for port 22 after 420 seconds, then poll the port every 5 seconds up to 1500 seconds(including the delay).

**Comment by torstenstern** [comment]
Hi,

is there any update on this?

**Comment by shinmog** [comment]
This will be fixed in v2.1.0. Use `panos_check` with an `initial_delay` of a minute or two.
